src/buffer/out/textBuffer.cpp

```diff
@@ -1145,7 +1145,7 @@ std::string TextBuffer::GenHTML(const TextAndColor& rows, const int fontHeightPo
                 }
 
                 const auto writeAccumulatedChars = [&](bool includeCurrent) {
-                    if (col > startOffset)
+                    if (col >= startOffset)

```

Comment:
FWIW, I'd love to know what this fixed... because this is in code that is part of Windows as well?

Reply:
I found a case where the last character of the text gets left out from the HTML.
It happens when the text colors of the last character and the character preceding it are different.

When we reach the last char and realize that the color is different, we do `writeAccumulatedChars(false)`. Which , at the end, sets `startOffset` to `col`. Here `col` is the index of the last char. After this, since we are at the last char, we do `writeAccumulatedChars(true)` which does nothing since `col > startOffset` is false.

Reply:
Excellent. Thanks! Good catch.